stl/message.py

```diff
@@ -45,7 +45,8 @@ class Message(stl.base.NamedObject):
     encoding: An stl.lib.Encoding object which will be used to serialize and
       deserialize MessageValues.
     is_array: Whether this message is an array.
-    fields: List of fields (stl.base.Field) defined in this protocol or message.
+    fields: Dictionary of fields (stl.base.Field) by name defined in this
+       protocol or message.
```

Comment:
How will you maintain the order of fields? The order will be important for byte stream protocols without field names.